praw/helpers.py

```diff
@@ -111,6 +115,119 @@ def valid_redditors(redditors, sub):
             if resp['ok']]
 
 
+def all_submissions(reddit_session,
```

Comment:
I'm a little on the fence about the name "all_submissions". Is there another name we can use that better represents the function? I feel that "all_submissions" only represents one possible use for this function.